source/release-notes/4-0-0.rst

```diff
@@ -18,57 +18,83 @@ New Features
     - Improved user management experience
     - Improved code editing
     - Improved configuration management
+    - Improved My Projects experience
     - And more...
 
   - New In-Context Editing system, Experience Builder (XB): |br|
 
     - Edit content in-place |br|
-    - Drag and drop components, assets to add/remove or rearrange your site (supports sites built with FreeMarker, React and can easily be extended to support Vue, Angular and other UI frameworks) |br|
+    - Drag and drop components, assets to add/remove or rearrange your project (supports projects
+      built with FreeMarker, React and can easily be extended to support Vue, Angular and other
```

Comment:
@rart is this still limited as stated or do we now directly support NextJS per https://craftercms.org/blog/2021/nextjs-and-craftercms-3-1